bounties_api/std_bounties/views.py

```diff
@@ -86,9 +86,9 @@ def post(self, request, bounty_id, fulfillment_id):
             fulfillment.fulfiller_review = review
         fulfillment.save()
         notification_client.rating_issued(
-            bounty.bounty_id, review, reviewer, reviewee)
+            bounty.bounty_id, review, fulfillment.id, reviewer, reviewee)
         notification_client.rating_received(
-            bounty.bounty_id, review, reviewer, reviewee)
+            bounty.bounty_id, review, fulfillment.id, reviewer, reviewee)
```

Comment:
@codeluggage this is still an issue - you need to combine this with a string or something else... Otherwise the fulfillment id will duplicate on both sides.

Reply:
It will get locked up on this line: `notification, created = Notification.objects.get_or_create(`
This needs to be more unique...